src/ControllerStatusIO.h

```diff
@@ -70,7 +70,7 @@ typedef struct
 {
     UINT16 interpolPeriod;                                  // Interpolation period of the controller
     int numGroup;                                           // Actual number of defined group
-    //int numRobot;                                         // Actual number of defined robot
+    int numRobot;                                           // Actual number of defined robot
```

Comment:
I don't think this is needed. The only place that I see it being used is to check if it's `>= 1`. 

It's not possible for this software to be running on a controller w/o a robot.